packages/coil-extension/src/background/services/BackgroundScript.ts

```diff
@@ -769,10 +769,11 @@ export class BackgroundScript {
     const id = this.assoc.getStreamId(frame)
     if (id) {
       this.log('resuming stream', id)
-      this.sendSetMonetizationStateMessage(
-        frame,
-        this.hasStreamPaid(frame) ? 'started' : 'pending'
-      )
+      // Always set "pending" transition state and emit event
```

Comment:
@sublimator Can you add an explanation (in the code) for why `pending` is always needed?

Reply:
I don't think it's *needed* per se (it's probably not) but who knows
This maintains the prior transitions, of which stopped->started wasn't part of